src/gnb/ngap/nnsf.cpp

```diff
@@ -11,11 +11,18 @@
 namespace nr::gnb
 {
 
-NgapAmfContext *NgapTask::selectAmf(int ueId)
+NgapAmfContext *NgapTask::selectAmf(int ueId, const int32_t &requestedSliceType)
 {
-    // todo:
-    for (auto &amf : m_amfCtx)
-        return amf.second; // return the first one
+    for (auto &amf : m_amfCtx) {
+        for (const auto &PlmnSupport : amf.second->plmnSupportList) {
```

Comment:
Please use lower-case for SingleSlice and SingleSlice. Because they are loop variables.

Reply:
I see. Will do that.

Reply:
Please rename `const auto &PlmnSupport` to `const auto &plmnSupport`

and 

`const auto &SingleSlice` to `const auto &singleSlice`

----------------------------------------------------

And please replace `const int32_t &requestedSliceType` to just `int32_t requestedSliceType`